waltz-service/src/main/java/com/khartec/waltz/service/survey/SurveyRunService.java

```diff
@@ -192,6 +194,8 @@ public void createSurveyInstancesAndRecipients(long surveyRunId,
                     }
                 }
         );
+
+        return instancesAndRecipientsToSave.size();
```

Comment:
is this the right thing to report ?   Looks like it's a number which may not accurately reflect what happened....